students/zach_thomson/lesson01/test_unit.py

```diff
@@ -0,0 +1,52 @@
+'''unit testing individual components of inventory management'''
+
+from unittest import TestCase
+#from unittest.mock import MagicMock
```

Comment:
In here, at the _inventory_management_ directory to the Python PATH, by doing the following:

```
import sys
sys.path.append("./inventory_management")
```

Note I'm using a UNIX-path, since I'm using a Mac, it might be a bit different if you're on a PC.

Reply:
That solved my issue - thanks so much for the help!